02 Algorithm Reference/05 Consolidating Data/01 Consolidating data/content.html

```diff
@@ -1,3 +1,7 @@
+<meta name="tag" content="consolidating data" />
+<meta name="tag" content="using data" />
+<meta name="tag" content="data event handlers" />
+<meta name="tag" content="tradebars event" />
```

Comment:
<meta name="tag" content="tradebars event" />
This isn't relevant here -- OnData(TradeBars data) is a core part of the API and not the consolidator system